wiki/rules.md

```diff
@@ -198,7 +197,7 @@ for the computed duration so that other systems do not need to compute it.
 If your start event, meaning the event that starts the chain of aggregation, 
 contains links that point to upstream events, HistoryRules makes it 
 possible for these to be aggregated into the object as well. They are 
-explained [**here**](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/history-rules.md)
+explained [**here**](history-rules.md).
 
```

Comment:
Line 208, 209 and 235 in this doc also need to updated I suppose.

Reply:
Will fix this in upcoming commit